store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -64,8 +64,16 @@
     private final PutMessageLock putMessageLock;
 
     public CommitLog(final DefaultMessageStore defaultMessageStore) {
-        this.mappedFileQueue = new MappedFileQueue(defaultMessageStore.getMessageStoreConfig().getStorePathCommitLog(),
-            defaultMessageStore.getMessageStoreConfig().getMapedFileSizeCommitLog(), defaultMessageStore.getAllocateMappedFileService());
+        if (defaultMessageStore.getMessageStoreConfig().isMultiCommitLogPathEnable()) {
+            this.mappedFileQueue = new MultiPathMappedFileQueue(defaultMessageStore.getMessageStoreConfig(),
+                defaultMessageStore.getMessageStoreConfig().getMapedFileSizeCommitLog(),
+                defaultMessageStore.getAllocateMappedFileService());
+        } else {
+            this.mappedFileQueue = new MappedFileQueue(defaultMessageStore.getMessageStoreConfig().getStorePathCommitLog(),
+                    defaultMessageStore.getMessageStoreConfig().getMapedFileSizeCommitLog(),
```

Comment:
No need to introduce isMultiCommitlogPathEnable.
Just check whether the old storePathCommitlog has path seperator ',' or not.